src/app/pages/not-found/not-found.component.html

```diff
@@ -0,0 +1 @@
+<p>not-found works!</p>
```

Comment:
Remove this default html tag from angular and replace it with a not found layout